homework/Factory Method/AutomobileDeveloper.cs

```diff
@@ -0,0 +1,20 @@
+﻿using FactoryMethod;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FactoryMethod
+{
+    public class AutomobileDeveloper : IDeveloper
+    {
+        public AutomobileDeveloper(string name) : base() { }
+
+        public IVehicle Create()
+        {
+            return new Automobile("Automobile");
```

Comment:
Why do you hardcode the name if you pass the name parameter when creating a class object?